Stripe/STPLocalizationUtils.h

```diff
@@ -21,3 +21,9 @@
 static inline NSString * _Nonnull STPLocalizedString(NSString* _Nonnull key, NSString * _Nullable __unused comment) {
     return [STPLocalizationUtils localizedStripeStringForKey:key];
 }
+
+/// Use to explicitly ignore static analyzer warning: "User-facing text should use localized string macro"
+__attribute__((annotate("returns_localized_nsstring")))
+static inline NSString * _Nonnull STPNonLocalizedString(NSString * _Nonnull string) {
```

Comment:
👍 

This is good also for other developers to know that strings are intentionally not translated (and not that they were accidentally missed)